src/general/cpuInfo.go

```diff
@@ -47,29 +47,51 @@ func NewCPULoad() *CPULoad {
 	return &CPULoad{}
 }
 
+// ReadCPULoad reads /proc/stat and returns the average load
+func ReadCPULoad() ([10]float64, error) {
+	data, error := os.ReadFile("/proc/stat")
+	if error != nil {
+		return [10]float64{}, error
+	}
+	string_data := string(data)
```

Comment:
`string_data` could be made `stringData` -> not wrong as such, but more like a Go good practice

Reply:
Got it!